packages/onboarding/src/scroll-to-top/index.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useLocation } from 'react-router-dom';
+
+type ScrollToTopProps = {
+	selector: string;
+};
+
+const ScrollToTop: React.FunctionComponent< ScrollToTopProps > = ( { selector } ) => {
+	const { pathname } = useLocation();
+
+	React.useEffect( () => document.querySelector( selector )?.scrollTo( 0, 0 ), [
```

Comment:
While I understand the flexibility of having a prop such as `selector` here, I have two suggestions:
- can we make it optional and when left undefined just use `window.scrollTo()`?
- for greater stability can we use a `ref` instead? ([example](https://levelup.gitconnected.com/react-tips-scroll-to-element-load-fonts-run-setstate-in-sequence-2d73c9f02ae1#d959))

Reply:
> can we make it optional and when left undefined just use `window.scrollTo()`?

Done ✅

> for greater stability can we use a `ref` instead? 

We can use a `ref` but that means that we have to pass it all the way from the parent's parent. We have to convert them to `React.forwardRef()` and I'm unsure what the consequences are. Besides that, we have to wrap the `<Router/>` in `<ScrollToTop/>` which is against best practices (see [react-router docs](https://reactrouter.com/web/guides/scroll-restoration/scroll-to-top)). 

To conclude: It becomes ugly pretty fast and we cannot oversee the consequences. 
We're going for the selector option w/ a fallback to `window.scrollTo()`.